infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -58,6 +58,14 @@ sites:
     plan: webmaster
     deploy_key: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIHvhy79hHjLcQJCcMNwci1Q/P/O2LwD4IzBVfkmRGKom
     <<: *webmaster-release-image-source
+  # BNF site.
+  bnf:
+    name: "Bibliotekernes Nationale Formidling"
+    description: "The BNF content sharing site"
+    primary-domain: bnf.dplplat01.dpl.reload.dk
```

Comment:
If the only domain used is the auto-generated domain, you shouldn't explicitly set a primary domain.

See example of going from autogenerated domains to live/prod domains here: https://github.com/danskernesdigitalebibliotek/dpl-platform/commit/3f3a7ea8830bc430dbbb0e024943312d35108543

Reply:
Dammit, I actually thought about removing it, but forgot to do it.